src/bundle/ui-dev/src/modules/multi-file-upload/components/upload-list/upload.item.component.js

```diff
@@ -575,18 +420,18 @@ export default class UploadItemComponent extends Component {
                 title={label}
                 tabIndex="-1"
             >
-                <Icon name="trash" extraClasses="ibexa-icon--small-medium" />
+                <Icon name="trash" extraClasses="ibexa-icon--small" />
             </div>
         );
     }
 
     render() {
-        const { uploaded, aborted, disallowedType, disallowedSize, failed, uploading, disallowedContentType, deleted, totalSize } =
-            this.state;
-        const isError = !uploaded && !aborted && (disallowedSize || disallowedType || disallowedContentType) && failed && !uploading;
+        const { failed, deleted, totalSize, errorMsgs, isMultipleErrosExpanded } = this.state;
+        const isMultipleErros = errorMsgs.length > 1;
```

Comment:
```suggestion
        const isMultipleErrors = errorMsgs.length > 1;
```
(but as above, maybe has instead of is?)